### PR TITLE
assorted 2.12 fixes

### PR DESCRIPTION
**Pull request by SethTisue**
fyi @eed3si9n on expecty addition

fyi @lihaoyi on utest's test suite being disabled

fyi @patriknw on Akka bump

fyi @bmc on weird grizzled/Wartremover thing

fyi @rossabaker on fs2 freeze

### CODE REVIEW

**Comment by SethTisue** [comment]
test run https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/2972/consoleFull

this will take a while, since I also recently bumped the 2.12 SHA so everything has to be rebuilt

**Comment by SethTisue** [comment]
newer test run: https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/2974/consoleFull

**Comment by SethTisue** [comment]
`failed: cats, scalaz8, play-core, akka-http-json, pprint`

* cats: out of memory error, probably intermittent. I'll add a commit giving it a bit more heap.
* scalaz8: looks like a transient failure to resolve a dependency.
* play-core: `Play-AHC-WS/compile` failed, which probably means we need to use an older play-ws
* akka-http-json: has a new `akka-http-jsoniter-scala` subproject. we need to either disable it, or add jsoniter to the build, which is probably a good idea.
* pprint: test failure, looks like probably an overly sensitive test case (`expected: String* = WrappedArray(empty iterator)`, `pprinted: String = <iterator>`) but could be a Scala regression

**Comment by SethTisue** [comment]
let's try again: https://scala-ci.typesafe.com/job/scala-2.12.x-integrate-community-build/2976/consoleFull

(I didn't address pprint yet)

**Comment by SethTisue** [comment]
fyi @marcospereira on the play-ws/playframework mismatch

fyi @plokhotnyuk on the jsoniter-scala addition

**Comment by SethTisue** [comment]
sigh

```
[play-core] [error] Failed tests:
[play-core] [error] 	play.libs.ws.WSSpec
```

I'll deal with that separately. I'm seeing enough green, on Jenkins and in my local run, to merge this.

**Comment by SethTisue** [comment]
dealt with the Play failure in 6361bd55c9644dd2ca1b19ec6525aa79b51589f4

**Comment by SethTisue** [comment]
pprint failure dealt with in 75a0295a15626448caede593d300ef014e186063

File: configs/community.dbuild [hunk 1]
```diff
@@ -1580,6 +1580,10 @@ build += {
     extra.options: ["-Dbintray.user=dummy", "-Dbintray.pass=dummy"]
     extra.commands: ${vars.default-commands} [
       "set every bintrayReleaseOnPublish := false"
+      // why we get wartremover failures in the community build that presumably
+      // don't happen in Brian's own CI, I have no idea. doesn't seem worth
+      // investigating
+      "set every wartremoverErrors := Seq()"
```
Comment (bmc) [inline configs/community.dbuild:1586]: Definitely not worth investigating. +1

**Pull request merged**